ReactWindows/ChakraBridge/pch.h

```diff
@@ -16,6 +16,28 @@
 
 #include <windows.h>
 
+extern "C" std::function<void(int, const wchar_t *)> g_loggingCallback;
+extern "C" bool g_isLoggingEnabled;
+
+#ifdef _DEBUG
+inline wchar_t* LogLevelToString(int logLevel)
+{
+    switch (logLevel)
+    {
+    case 0:
+        return L"Trace";
```

Comment:
For the way the code works now I didn't need, since the RnLog.Fatal calls Error underneath!
But you're touching an important point, the log output can have a [FATAL] prefix for this one. I'll do it for RnLog.Fatal calls.

For the current line I'm not sure, this is for JS originated lines, and they have what they have.

---
In reply to: [221402000](https://github.com/Microsoft/react-native-windows/pull/1987#discussion_r221402000) [](ancestors = 221402000)

Reply:
Ok, the output would show [CRITICAL], that's the value of the system provided enum.
Again, the change is in RnLog, not here.

---
In reply to: [221404461](https://github.com/Microsoft/react-native-windows/pull/1987#discussion_r221404461) [](ancestors = 221404461,221402000)